content/shared/v3-enterprise-get-started/_index.md

```diff
@@ -610,133 +624,165 @@ Usage: $ influxdb3 serve --writer-id=host01 --read-from-writer-ids=host02 --comp
+   ```
 
-# Example variables
-# writer-id: 'host02'
-# bucket: 'influxdb-3-enterprise-storage'
+   ```bash
+   ## NODE 2 — Writer/Reader Node #2
 
-Usage: $ influxdb3 serve --writer-id=host02 --compactor-id=c01 --read-from-writer-ids=host01 --object-store=s3 --bucket=influxdb-3-enterprise-storage --http-bind=0.0.0.0:8282 --aws-access-key-id=<AWS_ACCESS_KEY_ID> --aws-secret-access-key=<AWS_SECRET_ACCESS_KEY>
-```
+   # Example variables
+   # writer-id: 'host02'
+   # bucket: 'influxdb-3-enterprise-storage'
 
-For the compactor node, we need to set a few more options. First, we need to specify the mode which needs to be `--mode=compactor`; this ensures not only that it runs compaction, but that it *only* runs compaction. Since this node isn’t replicating data, we don’t pass it the replicas parameter, which means we need another way to tell it the hosts to run compaction. To do this, we set `--compaction-hosts` option with a comma-delimited list, similar to the replicas option. 
+   influxdb3 serve --writer-id=host02 --compactor-id=c01 --read-from-writer-ids=host01 --object-store=s3 --bucket=influxdb-3-enterprise-storage --http-bind=0.0.0.0:8282 --aws-access-key-id=<AWS_ACCESS_KEY_ID> --aws-secret-access-key=<AWS_SECRET_ACCESS_KEY>
+   ```
 
-```
-## NODE 3 — Compactor Node
+2. Start the dedicated compactor node, which uses the following options:
 
-# Example variables
-# writer-id: 'host03'
-# bucket: 'influxdb-3-enterprise-storage'
-# compactor-id: 'c01'
+   - `--mode=compactor`: Ensures the node **only** runs compaction.
+   - `--compaction-hosts`: Specifies a comma-delimited list of hosts to run compaction for.
+  
+  _**Don't include the replicas (`--read-from-writer-ids`) parameter because this node doesn't replicate data._
 
-Usage: $ influxdb3 serve --writer-id=host03 --mode=compactor --compactor-id=c01 --compaction-hosts=host01,host02 --run-compactions --object-store=s3 --bucket=influxdb-3-enterprise-storage --aws-access-key-id=<AWS_ACCESS_KEY_ID> --aws-secret-access-key=<AWS_SECRET_ACCESS_KEY>
-```
+   ```bash
 
-### 
+   ## NODE 3 — Compactor Node
 
-### High Availability with Read Replicas and a Dedicated Compactor
+   # Example variables
+   # writer-id: 'host03'
+   # bucket: 'influxdb-3-enterprise-storage'
+   # compactor-id: 'c01'
 
-To create a very robust and effective setup for managing time-series data, we recommend running ingest nodes alongside read-only nodes, and leveraging a compactor-node for excellent performance. 
+   influxdb3 serve --writer-id=host03 --mode=compactor --compactor-id=c01 --compaction-hosts=host01,host02 --run-compactions --object-store=s3 --bucket=influxdb-3-enterprise-storage --aws-access-key-id=<AWS_ACCESS_KEY_ID> --aws-secret-access-key=<AWS_SECRET_ACCESS_KEY>
```

Comment:
Not suggestion a change here, just noting that when commands force a user to scroll (in full-width view), I do multiline commands with each flag on its own line and the newlines escaped.

Reply:
> Not suggestion a change here, just noting that when commands force a user to scroll (in full-width view), I do multiline commands with each flag on its own line and the newlines escaped.

Def, thanks! Too many other annoyances clouding my vision